src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/dto/LegacyReason.kt

```diff
@@ -6,18 +6,22 @@ package uk.gov.justice.digital.hmpps.hmppsnonassociationsapi.dto
  * Values taken from Prison API `references_codes` table: https://github.com/ministryofjustice/prison-api/blob/acb4076b98c706c49edd61f21061c84eb6a837bf/src/main/resources/db/migration/data/R__2_3__REFERENCE_CODES.sql#LL5181-L5184
  */
 enum class LegacyReason(val description: String) {
-  BULLYING("Anti Bullying Strategy"),
-  PERPETRATOR("Perpetrator"),
-  RIVAL_GANG("Rival Gang"),
-  VICTIM("Victim"),
+  BUL("Anti Bullying Strategy"),
+  PER("Perpetrator"),
+  RIV("Rival Gang"),
+  VIC("Victim"),
+  NOT_RELEVANT("Not relevant"),
+  UNKNOWN("Unknown"),
```

Comment:
are we assuming that we can add these so that our sync _to_ NOMIS involves no data loss?

Reply:
yes ideally